crates/biome_js_analyze/src/analyzers/style/use_numeric_literals.rs

```diff
@@ -134,8 +148,16 @@ impl CallInfo {
     }
 
     fn to_numeric_literal(&self) -> Option<JsSyntaxToken> {
-        i128::from_str_radix(&self.text, self.radix as u32).ok()?;
-        let number = format!("{}{}", self.radix.prefix(), self.text);
+        // `parseInt` ignores leading and trailing white-spaces
+        let text = self.text.trim();
```

Comment:
Can't we use [`text_trimmed`](https://github.com/biomejs/biome/blob/599665712c66c5e72198193ce926770c2b838507/crates/biome_rowan/src/syntax/node.rs#L121)?